recipe/meta.yaml

```diff
@@ -15,17 +15,21 @@ source:
   sha256: 00a65114d697cfaa8fe0630281d76fd1b77afcd95cd5e40ec6a02cbbadbfea71
 
 build:
-  number: 4
+  number: 5
```

Comment:
```suggestion
  number: 5
  track_features:
     - sqlite_with_icu   # [with_icu == "yes"]
```

Reply:
Also use a jinja var for build number and 100 for without icu build.

Reply:
> Also use a jinja var for build number and 100 for without icu build.

Something like this?

```
build:
  number: {{ build_number if with_icu else build_number + 100 }}
```